docs/compose/Dialog/README.md

```diff
@@ -39,7 +39,7 @@ main [Readme](https://github.com/skyscanner/backpack-android#installation) for a
 Example of a success dialog with three buttons
 
 ```Kotlin
-import net.skyscanner.backpack.compose.dialog.BpkDialog
+import net.skyscanner.backpack.compose.dialog.BpkSuccessDialog
```

Comment:
Can you also include screenshots for the image dialog at the top in the screenshot section please?